pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "klaam"
+version = "0.0.1"
+description = "Arabic speech recognition, classification and text-to-speech."
+license = "MIT"
+authors = [
+    "Zaid Alyafeai <alyafey22@gmail.com>",
+    "mustafa0x"
+]
+readme = 'README.md'
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
@zaidalayfeai, I think most of the code runs on google colab. Google colab currently runs on python3.7 right?

Reply:
I couldn't run the project using `python 3.7.13` as some packages require new versions. Google Colab does use `python 3.7.13`, maybe we do need to resolve some conflicts with packages to be able to use `python 3.7.13`.

Reply:
If Python3.8 is covered by most LTS Linux OSes ( I am assuming users will have Linux LTS in their machines) then this should be fine. Debian 11 supports Python3.9, Ubuntu 20.04 supports Python 3.8. I think it should be fine to stick with Python 3.8 and onwards. Google colab will update soon